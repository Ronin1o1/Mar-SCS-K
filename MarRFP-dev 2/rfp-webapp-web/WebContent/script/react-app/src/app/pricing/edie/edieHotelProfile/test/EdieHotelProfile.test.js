import React from "react";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";
import EdieHotelProfileList from "../content/EdieHotelProfileList";
import FileUpload from './../../../../shared/components/fileupload';

describe("EdieHotelProfileList", () => {
  it("renders without crashing", () => {
    mount(<EdieHotelProfileList />);
  });

  it("renders correctly", () => {
    const tree = mount(<EdieHotelProfileList />);
    expect(toJson(tree)).toMatchSnapshot();
  });

  it("renders FileUpload without crashing", () => {
    mount(<FileUpload />);
  });

  it("renders FileUpload correctly", () => {
    const tree = mount(<FileUpload />);
    expect(toJson(tree)).toMatchSnapshot();
  });

});

