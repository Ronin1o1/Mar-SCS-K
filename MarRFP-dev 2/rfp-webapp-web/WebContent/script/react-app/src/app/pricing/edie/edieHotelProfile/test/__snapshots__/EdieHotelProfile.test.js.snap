// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EdieHotelProfileList renders FileUpload correctly 1`] = `
<FileUpload>
  <div
    className="gridHeaderFileUpload"
  >
    <label
      className="emailLabel"
    >
      Select a file to attach with email
    </label>
    <input
      id="emailFileUpload"
      name="emailFileUpload"
      onChange={[Function]}
      type="file"
    />
    <input
      id="clear"
      name="Clear"
      onClick={[Function]}
      type="button"
      value="Clear"
    />
  </div>
</FileUpload>
`;
